packages/airnode-validator/templates/0.2/receipt.json

```diff
@@ -51,7 +51,8 @@
     "nodeVersion": {
       "__regexp": "^0\\.2\\.[0-9]+$",
       "__catch": {
-        "__level": "warning"
+        "__level": "error",
```

Comment:
Are we sure we want to change this to error now?

Reply:
I will revert this from the versions below 0.4 as I do not see why they should be changed.